### PR TITLE
KOGITO-3828 - dmn-drools-metrics-quarkus example update

### PR DESCRIPTION
**Pull request by r00ta**
Jira: https://issues.redhat.com/browse/KOGITO-3828

This pr aims to update the examples according to the refactoring of the monitoring addon proposed here: https://github.com/kiegroup/kogito-runtimes/pull/890

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by r00ta** [comment]
Jenkins retest this please

File: kogito-travel-agency/extended/visas/src/main/resources/application.properties [hunk 1]
```diff
@@ -45,3 +45,5 @@ mp.messaging.outgoing.kogito-usertaskinstances-events.value.serializer=org.apach
 mp.messaging.outgoing.kogito-variables-events.connector=smallrye-kafka
 mp.messaging.outgoing.kogito-variables-events.topic=kogito-variables-events
 mp.messaging.outgoing.kogito-variables-events.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+
+kogito.addon.monitoring.process.deafult.bean.create=false
```
Comment (cristianonicolai) [inline kogito-travel-agency/extended/visas/src/main/resources/application.properties]: @r00ta why not have the process metrics here in this example?

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/DroolsMetricsTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.containsString;
+
+@QuarkusTest
+public class DroolsMetricsTest {
+
+    @Test
+    public void testDrlMetrics() {
+        given()
```
Comment (cristianonicolai) [inline dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/DroolsMetricsTest.java:30]: @r00ta I think these type of test should be in the integration test module from kogito-runtimes, one for process and another for rules, would be quite valuable.

File: kogito-travel-agency/extended/visas/src/main/resources/application.properties (hunk 1 above)
Comment (r00ta) [reply kogito-travel-agency/extended/visas/src/main/resources/application.properties]: Hi @cristianonicolai , in this example a custom `VisaApplicationPrometheusProcessEventListener` is provided by the user, that's why with this config property we disable the creation of the default bean

File: dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/DroolsMetricsTest.java (hunk 2 above)
Comment (r00ta) [reply dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/DroolsMetricsTest.java:30]: Makes sense, I'll add it

Comment (r00ta) [reply dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/DroolsMetricsTest.java:30]: added to the `kogito-travel-agency/extended/travels` example

**Comment by r00ta** [comment]
Jenkins retest this please

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

**Review by MarianMacik — APPROVED** [review]
LGTM

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Hi @cristianonicolai , are you ok to move forward with this?

File: dmn-drools-quarkus-metrics/README.md [hunk 3]
```diff
@@ -13,33 +13,20 @@ You will need:
...
 ### How to enable the feature
 
-Import the following dependency in your `pom.xml`:
+Import the following dependency in your `pom.xml` if you are using quarkus:
 
 ```XML
 <dependency>
  <groupId>org.kie.kogito</groupId>
- <artifactId>monitoring-prometheus-addon</artifactId>
+ <artifactId>monitoring-prometheus-quarkus</artifactId>
```
Comment (cristianonicolai) [inline dmn-drools-quarkus-metrics/README.md]: @r00ta I think it is fine to have -quarkus but would still make sense to keep suffix -addon.
```suggestion
 <artifactId>monitoring-prometheus-quarkus-addon</artifactId>
```

Comment (r00ta) [reply dmn-drools-quarkus-metrics/README.md]: Hi @cristianonicolai , do you think we should rename everything in `kogito-runtimes` as well? `artifactId` + `folder names`? 
Screenshot from 2020-12-09 09-10-38

Comment (r00ta) [reply dmn-drools-quarkus-metrics/README.md]: Forget the elastic modules.. these are for the next pr

Comment (cristianonicolai) [reply dmn-drools-quarkus-metrics/README.md]: @r00ta I usually prefer to have artifactId = folder name, but that's just my personal preference, although that seems to be the case for most of our modules. I don't think you need to change the _common_ ones, but would be nice to keep consistent that only modules that are "user facing" end with _-addon_. So I guess not only adding _-addon_ to _-quarkus_ and _-springboot_, but also removing _-addon_ from _monitoring-addon_ and _monitoring-prometheus-addon_.

Comment (r00ta) [reply dmn-drools-quarkus-metrics/README.md]: @cristianonicolai here's the new modules, let me know if we are good to go once the CI is green
Screenshot from 2020-12-10 09-37-36

Comment (cristianonicolai) [reply dmn-drools-quarkus-metrics/README.md]: thanks @r00ta!

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**